src/shared/components/Header/styles.ts

```diff
@@ -1,11 +1,17 @@
 import styled from 'styled-components';
 import media from 'shared/media-types';
 
-export const HeaderWrapper = styled.header`
+type StyleProps = {
+  isOpenMenu: boolean;
+};
+
+export const HeaderWrapper = styled.header<StyleProps>`
   position: relative;
   display: flex;
   align-items: center;
   height: 80px;
+  ${({ isOpenMenu }: StyleProps): string => `background: ${isOpenMenu ? '#fff' : 'transparent'};`};
```

Comment:
please do not use `#fff` --> `${Colors.WHITE}`

Reply:
Done!